harvest_cli/ocua.py

```diff
@@ -7,7 +7,8 @@
 import redis
 from os import getenv
 from pyhorn.endpoints.search import SearchEpisode
-
+from geolocation import Geolocate
+from user_agents import parse
```

Comment:
for clarity let's change this to `import user_agents` here and `user_agents.parse()` in the function code